Dockerfile

```diff
@@ -1,12 +1,18 @@
 # syntax=docker/dockerfile:1
-ARG PYTHON_VERSION=3.11
+ARG PYTHON_VERSION=3.12
+ARG PLONE_VERSION=6.1.0rc2
```

Comment:
This was previously passed in from the GHA based on version.txt, so we only needed to update the version in one place. Is there a reason that won't work any longer?

Reply:
It is still passed in as --build-arg when we call buildx from the Makefile. 

I added the default values to silence a warning from buildx that a variable is used in the dockerfile that can be empty if it isn't passed in as a --build-arg :-P

Reply:
@fredvd Well, we could just leave the warning there since we were doing it intentionally. I'm worried that we'll miss updating all copies of the version when we make updates in the future.